CHANGELOG.md

```diff
@@ -46,6 +46,7 @@ Unless otherwise specified, any version comparison below is the comparison of se
   using `/fill` command to fill a large area).
 - Introduced `ChunkSectionLocks`, which replaced the old `StampedLock` in `Chunk`. Instead of locking the whole chunk when reading/writing
   blocks/biomes, only the related chunk section will be locked now. This should improve the performance of chunk reading/writing.
+- Implemented flower pot, and add custom block tag `allay:pottable_plant` which mark thant the plant can be potted.
```

Comment:
This change should be annotated with (API) prefix (you changed BlockCustomTags.java which is in api module)